### ISSUE TITLE
bug with scrolling to found text

### ISSUE THREAD

**Issue by cser** [Proektsoftbg/Calcpad#89]
When I try to find "TEXT" selection works correctly, but scrolling is strange - TEXT is always outside the screen.
```
'a| ← cursor position in case #1
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'TEXT
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a
'a| ← cursor position in case #2
```

**Comment by Proektsoftbg** [comment]
@cser Thank you for reporting this. It is now fixed in the [v5.8.9](https://github.com/Proektsoftbg/Calcpad/releases/tag/v5.8.9) release.

**Comment by cser** [comment]
Thanks, Ned! Now scrolling works correctly.
